<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectPropertiesWindow</class>
 <widget class="QDialog" name="ProjectPropertiesWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>140</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>140</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>140</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Project Properties</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name</string>
       </property>
       <property name="buddy">
        <cstring>lineEditName</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditName">
       <property name="placeholderText">
        <string>Project Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelAuthor">
       <property name="text">
        <string>Author</string>
       </property>
       <property name="buddy">
        <cstring>lineEditAuthor</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditAuthor">
       <property name="placeholderText">
        <string>Author</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelLanguage">
       <property name="text">
        <string>Language</string>
       </property>
       <property name="buddy">
        <cstring>languagePicker</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="novelist::LanguagePicker" name="languagePicker"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>novelist::LanguagePicker</class>
   <extends>QComboBox</extends>
   <header>widgets/LanguagePicker.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProjectPropertiesWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProjectPropertiesWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
